plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/task/MetalavaSignature.kt

```diff
@@ -6,6 +6,39 @@ import org.gradle.api.Project
 import org.gradle.api.tasks.JavaExec
 import org.gradle.api.tasks.TaskProvider
 import java.io.File
+import java.util.Locale
+
+private val sourceLanguageDirectoryNames = listOf("java", "kotlin")
```

Comment:
A little bit of a nit, but since these are all `private` anyways, can we move it into `MetalavaSignature`? That way we won't produce the empty `MetalavaSignatureKt` class in the signature file.